examples/favicon-cs/README.md

```diff
@@ -0,0 +1,16 @@
+## Fetching a favicon in a content script
+
+This example fetches the favicon from www.google.com and inserts it at the top left of every page.
+
+Note: This extension does not work on `chrome://extensions`.
+
+See [Fetching favicons](https://developer.chrome.com/docs/extensions/mv3/favicon) to learn more. 
+
+## Testing the extension
+
+1. Follow the instructions to load an [unpacked extension](https://developer.chrome.com/docs/extensions/mv3/getstarted/development-basics/#load-unpacked).
+2. Navigate to [www.example.com](https://www.example.com/).
+
+It should look like this:
+
+<img src="https://wd.imgix.net/image/BhuKGJaIeLNPW9ehns59NfwqKxF2/3Q1glvnzbWhraXRtnGOy.png" alt="Content script using the Favicon API" width="400"/>
```

Comment:
It looks like this is our first time using imgix, are we comfortable hosting there?

Reply:
Oh right, I forgot to mention this is where all images of https://developer.chrome.com/ are hosted.
I don't think this is a blocker, but it's probably something we should agree on with the team. Thanks for bringing this up 👍